tests/network/vmi_multus.go

```diff
@@ -258,6 +271,30 @@ var _ = Describe(SIG("Multus", Serial, decorators.Multus, func() {
 				By("checking virtual machine instance can ping using ptp cni plugin")
 				Expect(libnet.PingFromVMConsole(detachedVMI, multiIfaceNetGatewayIP2)).To(Succeed())
 			})
+
+			It("VirtualMachineInstance with cni ptp plugin interface with custom MAC address", func() {
+				By("Verifying the desired custom MAC is the one that was actually configured on the interface.")
+				ipLinkShow := fmt.Sprintf("ip link show eth2 | grep -i \"%s\" | wc -l\n", customMacAddress)
```

Comment:
lets make please eth2 const, because if someone change the order of eth1..N above it can be mixed